linalg/densemat.hpp

```diff
@@ -524,12 +524,49 @@ void AddMult_a_VWt(const double a, const Vector &v, const Vector &w,
 void AddMult_a_VVt(const double a, const Vector &v, DenseMatrix &VVt);
 
 
-/** Class that can compute LU factorization of external data and perform various
+/** Abstract class that can compute factorization of external data and perform various
     operations with the factored data. */
-class LUFactors
+class Factors
 {
 public:
+
    double *data;
+
+   Factors() { }
```

Comment:
Documentation?

Reply:
I will wait to make changes in this file until https://github.com/mfem/mfem/pull/2887 is merged.